### PR TITLE
Updated API Link

### PR DESCRIPTION
**Pull request by mainboiii**
Updated Repeats API link as we now have a new one.

### CODE REVIEW

**Review by ririxidev — CHANGES_REQUESTED** [review]
bump version & upload proof of working presence :)

**Comment by mainboiii** [comment]
What do you mean by bump version? soory lol

**Comment by mainboiii** [comment]
Will get a screenshot soon for you.

File: websites/R/Repeat/presence.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ let sartist, strack, slisteners, sdj;
 
 function newStats(): Promise<void> {
   window
-    .fetch("https://stats.repeat.pw")
+    .fetch("https://repeat.bozzy.online/api/nowplaying/1")
```
Comment (Bas950) [inline websites/R/Repeat/presence.ts:6]: You may only make HTTP/HTTPS requests to `premid.app` or the presence website API. If you are using external domains, you will be required to explain why it is necessary.

bozzy.online is not the presence website, so you need an api from repeat.pw or repeatradio.net

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by mainboiii** [comment]
Here is the PreMiD precence working:
xebYu0DR

**Review by Bas950 — APPROVED** [review]

**Review by mainboiii — COMMENTED** [review]
Bump Version and showed screenshot and fixed other isuues.

**Review by Timeraa — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]

**Pull request merged**